simvue/run.py

```diff
@@ -209,28 +210,6 @@ def __init__(
             )
             else self._user_config.metrics.emission_metrics_interval
         )
-        if mode == "offline":
```

Comment:
Why is the offline mode not being included?

Reply:
This is not the removal of offline mode but both online and offline in the class `__init__`, instead this is initialised to `None` as CO2 emission metrics are off by default. Assignment to a `CO2Monitor` instance occurs within the `config(...)` method.